generic-sensor/generic-sensor-tests.js

```diff
@@ -251,22 +301,22 @@ function runGenericSensorTests(sensorType) {
   }, `${sensorType.name}: sensor receives suspend / resume notifications when\
   cross-origin subframe is focused`);
 
-  test(() => {
-     assert_throws("NotSupportedError", () => { new sensorType({invalid: 1}) });
-     assert_throws("NotSupportedError", () => { new sensorType({frequency: 60, invalid: 1}) });
-     if (spatialSensors.indexOf(sensorType.name) == -1) {
-       assert_throws("NotSupportedError", () => { new sensorType({referenceFrame: "screen"}) });
-     }
-  }, `${sensorType.name}: throw 'NotSupportedError' for an unsupported sensor option`);
+//  Re-enable after https://github.com/w3c/sensors/issues/361 is fixed.
+//  test(() => {
+//     assert_throws("NotSupportedError", () => { new sensorType({invalid: 1}) });
+//     assert_throws("NotSupportedError", () => { new sensorType({frequency: 60, invalid: 1}) });
+//     if (spatialSensors.indexOf(sensorType.name) == -1) {
+//       assert_throws("NotSupportedError", () => { new sensorType({referenceFrame: "screen"}) });
+//     }
+//  }, `${sensorType.name}: throw 'NotSupportedError' for an unsupported sensor option`);
 
   test(() => {
     const invalidFreqs = [
       "invalid",
       NaN,
       Infinity,
       -Infinity,
-      {},
-      undefined
```

Comment:
Why remove the `undefined` value?

Reply:
Right, unlike the following test case for `referenceFrame`, the `frequency` field does not have a default value, so `undefined` should be returned here. Thanks for the catch!